<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShaCheck</class>
 <widget class="QMainWindow" name="ShaCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>195</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ShaCheck</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushBtnSelectFile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pick file...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFilename">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelHash">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hash:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHash">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Paste cryptographic hash here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnCheck">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check!</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMatch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelLabelMatch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Match:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLabelFilename">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLabelCalculated">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>47</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCalculated">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
